pkg/3scale/amp/template/adapters/system.go

```diff
@@ -161,6 +165,52 @@ func (s *System) options() (*component.SystemOptions, error) {
 	sob.BackendSharedSecret("${SYSTEM_BACKEND_SHARED_SECRET}")
 	sob.TenantName("${TENANT_NAME}")
 	sob.WildcardDomain("${WILDCARD_DOMAIN}")
-	sob.StorageClassName(nil)
+
+	s.setFileStorageOptions(&sob)
 	return sob.Build()
 }
+
+func (s *System) setFileStorageOptions(sob *component.SystemOptionsBuilder) {
```

Comment:
In templates we now set some options conditionally depending on the desired storage type for system, which changes depending on the template being used